frontend/src/components/BackupEditDialog.vue

```diff
@@ -98,6 +98,15 @@ const closeDialog = () => {
   emits('close');
 };
 
+watch(
+  ()=>viewDialog.value,
+  (value) => {
+    if (!value) {
+      closeDialog();
```

Comment:
```suggestion
      emits('close');
```

Reply:
closeDialog is used for button "close"
`<v-btn class="ms-auto" text="Close" @click="closeDialog"></v-btn>`